Makefile

```diff
@@ -19,7 +19,6 @@ deb:
 		python-dbus \
 		python-nacl \
 		python-requests-oauthlib \
-		python-configparser \
 		python-future \
```

Comment:
deze dependencies kunnen ook weg of niet?

Reply:
Fixed